docs/legal.md

```diff
@@ -57,6 +57,11 @@ Following data gets stored from the bot:
 - `channelid` reference to the channel with the verify message
 - `messageid` reference to the verify message
 - `language` language setting for the server
+- `autoVerify` automatically verify when joining a server
+- `autoAddUnverified` automatically add the unverified role to every new member of the server
+- `verifyMessage` message to show to the user when verifying
+- `logChannel` channel to log the emails of verified users
```

Comment:
Yeah, that is true, but I don't store these emails and the user gets warned if the log channel is enabled. It is stored on the server and therefore the server owner has to delete the email if the user requests that. This feature was only implemented because some people needed access to the plain text email addresses for some reason. I am not really a fan of it, but I think the hashing won't be a solution...